django/test/runner.py

```diff
@@ -422,6 +448,9 @@ def __init__(self, pattern=None, top_level=None, verbosity=1,
         self.parallel = parallel
         self.tags = set(tags or [])
         self.exclude_tags = set(exclude_tags or [])
+        self.pdb = pdb
+        if self.pdb and self.parallel > 1:
+            raise ValueError("You cannot use --pdb with parallel tests; pass --parallel=1 to use it.")
```

Comment:
OK, quick question: should we just automatically set `self.parallel=1` if you pass `--pdb`?

Reply:
Grrr. Single quotes. I’ll change that.

(When is Black stable?)